file-formats/gsmg/examples/z_aff_example_gsmg.gsmg.json

```
@@ -0,0 +1,12 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "For testing",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
+  },
+  "generalInformation": {
+    "type": "USAGE",
+    "detailedDescription": "This Metric Group is for testing"
+  }
+}
```

Just add an empty line at the end-of-file
```suggestion
}

```